org.freedesktop.LinuxAudio.Plugins.ChowTapeModel.json

```diff
@@ -2,7 +2,7 @@
     "id": "org.freedesktop.LinuxAudio.Plugins.ChowTapeModel",
     "branch": "21.08",
     "runtime": "org.freedesktop.LinuxAudio.BaseExtension",
-    "runtime-version": "21.08",
+    "runtime-version": "stable",
```

Comment:
what exactly this means?
so for example plugin built against 21.08 sdk can be used in future 22.08 runtime?

Reply:
this is the base app. It's a recent change and should be that way.

Reply:
> this is the base app. It's a recent change and should be that way.

great news :)